frontend/src/index.jsx

```diff
@@ -39,6 +40,22 @@ const fetchReportData = async (dialect) => {
   return parseReportData(lines);
 };
 
+const fetchAllReportData = async () => {
+  let loaderData = {};
+  const fetchPromises = Object.keys(dialectToName).map(async (draft) => {
```

Comment:
This code duplicates a lot from the function above. Why not do something like this:
```js
Object.keys(dialectToName).map(dialect => fetchReportData(dialect))
```
?
Probably changing document title should be moved to some different place then

Reply:
Done!

> Probably changing document title should be moved to some different place then

This shouldn't cause any problems since in implementation-specific pages, I've overwritten the `title` with the implementation's name.